judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/AssociationRelationshipsTest.java

```diff
@@ -84,15 +96,15 @@ public String getModelName() {
             "REQ-ENT-012"
     })
     public void testSetUnsetSingleRelation() {
-        assertEquals(Optional.empty(), entityADao.getSingleConA(entityA));
+        assertEquals(Optional.empty(), entityADao.querySingleConA(entityA));
```

Comment:
If you renamed getRelationName in cases with cardinality 0..1, why didn't you rename it in cases with cardinality 0..*? See the 229th and  230th row this file!